### DISCUSSION TITLE
interaction with the cards

### DISCUSSION THREAD

**Discussion by YuhangGu** [prabhuignoto/react-chrono#128] [Ideas]
Dear Prabhu, I like this project and currently working on an event timeline project. I'm wandering is it possible to get the Item index when clicked on the corresponding card? I hope to update a map that is displaying locations of the items.

**Comment by prabhuignoto** [comment]
use `onItemSelected` prop to access the selected or active index. payload of the passed data looks like below

```
{
    cardDetailedText: "On 10 May 1940, Hitler began his long-awaited offensive in the west"
    ​cardSubtitle: "Men of the British Expeditionary Force (BEF) wade out to a destroyer during the evacuation from Dunkirk."
    ​cardTitle: "Dunkirk"
    ​index: 0
    ​title: "May 1940"
}
```
